.travis.yml

```diff
@@ -17,6 +17,7 @@ before_script:
   - sudo service postgresql stop
   - docker-compose up -d
   - cp ormconfig.travis.json ormconfig.json
+  - npm install sqlite3 --build-from-source
```

Comment:
why sqlite? shouldn't docker-compose.yml handle this?

Reply:
`sqlite` tests are not run inside a `docker` container, so `docker-compose` isn't used for it.

This command has to be run because the pre-built versions of `sqlite3` were not compiled with a recent enough `sqlite` lib, as in, `>= 3.24.0`. When you install `sqlite3` from source, it builds using the bundled version that goes with it, `3.24.0`. or with a custom version that can be specified with the `--sqlite` flag.

Reply:
I missed it installs npm package. thanks for the explanation!